yarn.lock

```diff
@@ -1401,19 +1401,6 @@
     "@babel/helper-module-imports" "^7.10.4"
     "@rollup/pluginutils" "^3.1.0"
 
-"@rollup/plugin-commonjs@^19.0.0":
```

Comment:
I don't know why the yarn.lock changed when I installed the dependencies. I can revert these changes if it's a problem